testsuite/pytests/test_aeif_cond_alpha_multisynapse.py

```diff
@@ -147,12 +149,13 @@ def test_single_multi_synapse_equivalence(self, have_plotting):
                 _ax.legend()
 
             ax[-1].semilogy(multisynapse_neuron_vm.get("events")["times"], error, label="errror")
-            fig.savefig("test_aeif_cond_alpha_multisynapse.png")
+            fig.savefig(os.path.join(os.environ.get("REPORTDIR", ""), "test_aeif_cond_alpha_multisynapse.png"))
 
         # compare with a large tolerance because previous PSPs affect subsequent PSPs in the multisynapse neuron
         np.testing.assert_allclose(error, 0, atol=1e-6)
 
     def test_recordables(self):
+        r"""Test that the right number of recordables are created when setting ``record_from``."""
```

Comment:
Multi-line literal for single line string. No need to be raw either.
```suggestion
        "Test that the right number of recordables are created when setting ``record_from``."
```

Reply:
I wonder why this is not caught by the code formatter… :thinking: